public/home/section_add_ons_extend_functionality.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: "Add-ons Extend Functionality"
+title: "Extensions"
 image: "add_ons.png"
 video: "OANsA6fMJKg?start=6"
 weight: 10
 ---
 
-Use various add-ons available within Orange to mine data from external data sources, perform natural language processing and text mining, conduct network analysis, infer frequent itemset and do association rules mining. Additionally, bioinformaticians and molecular biologists can use Orange to rank genes by their differential expression and perform enrichment analysis. Check out also Orange cousins [Single Cell](https://singlecell.biolab.si/) and [Quasar](https://quasar.codes/).
\ No newline at end of file
+Specialized add-ons can access external data sources, perform natural language processing and text mining, conduct network analysis, do association rules mining, or address fairness in machine learning. Some address specialized data, such as time series, survival data sets, spectra, or gene expressions. Also, check out Orange's cousin [Quasar](https://quasar.codes/).
```

Comment:
Is this "can access external data sources" necessary and important ... or is it just misleading because one might think that Orange widgets cannot access non-local data sources?

Reply:
Hmm, true. I'll delete that part.